### PR TITLE
chore: update README and supported versions [CI SKIP]

### PR DESCRIPTION
**Pull request by bk201**
Add v1.3.1 and refresh screenshots.

Depends on https://github.com/harvester/docs/pull/598.

### CODE REVIEW

**Review by ibrokethecloud — APPROVED** [review]
lgtm. thanks.

File: README.md [hunk 1]
```diff
@@ -88,13 +88,15 @@ During the installation, you can either choose to **create a new Harvester clust
 > - __\<version\>*__ means the release branch is under active support and will have periodic follow-up patch releases.
 > - __Latest__ release means the version is the latest release of the newest release branch.
 > - __Stable__ release means the version is stable and has been widely adopted by users.
+> - __EOL__ release means the version is end of its life.
```
Comment (jillian-maroket) [inline README.md]:
```suggestion
> - __EOL__ means that the software has reached the end of its useful life and no further code-level maintenance will be provided. You may continue to use the software within the terms of the licensing agreement.
```

Comment (jillian-maroket) [reply README.md]: I lifted parts of the new wording from the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-3-x/).

**Review by jillian-maroket — COMMENTED** [review]
PTAL at the suggested definition for "EOL".

File: README.md (hunk 1 above)
Comment (bk201) [reply README.md]: Thanks, fixed.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**